### PR TITLE
Test port to rattler-build

### PR DESCRIPTION
**Pull request by wolfv**

### CODE REVIEW

File: recipe/recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  summary: Modern Fortran Parser
+  documentation: https://github.com/stfc/fparser
+  repository: https://github.com/stfc/fparser
+
+extra:
+  recipe-maintainers:
+    - arporter
+    - matthewrmshin
+    - MatthewHambley
+    - rupertford
```
Comment (arporter) [inline recipe/recipe.yaml]: Please could you remove Rupert Ford from this list. He is sadly no longer with us :-(

**Review by arporter — CHANGES_REQUESTED** [review]
Hi @wolfv, thanks for this. Just one small update and I see that the CI is failing at the moment.

File: recipe/recipe.yaml (hunk 1 above)
Comment (wolfv) [reply recipe/recipe.yaml]: I have just done this, as requested.

**Comment by wolfv** [comment]
Closing this as the recipe translation is unrelated to the other problem regarding SETUPTOOLS_SCM_VERSION.

**Pull request closed without merging**